docs.bruno/Testing - Get Data.bru

```diff
@@ -0,0 +1,11 @@
+meta {
+  name: Testing - Get Data
+  type: http
+  seq: 1
+}
+
+get {
+  url: https://api.github.com/users/usebruno
+  body: none
+  auth: none
```

Comment:
Consider adding headers to the HTTP request if necessary, for example to specify the content type or to include authorization tokens. [medium]